src/main/java/bob/DeleteCommand.java

```diff
@@ -0,0 +1,57 @@
+package bob;
+
+/**
+ * Represents a command to delete a task from the task list by its index.
+ * This command removes a specified task from the list.
+ */
+public class DeleteCommand extends Command {
+    protected static String[] params = new String[] {
+        "delete"
+    };
+    protected static int paramCount = 1;
+    protected static String identifier = "delete";
```

Comment:
Since `identifier` is a constant, it should be in uppercase, `IDENTIFIER` according to the coding standard for this course!